apps/data-scanner/lib/api/scanner/app.mjs

```diff
@@ -1,15 +1,19 @@
 import express from 'express';
+import { updateSessions } from '../../utils/updateSessions.mjs';
 import { Api } from 'telegram';
-import { CronJob } from "cron";
+import { CronJob } from 'cron';
 import { Sessions } from '../../models/index.mjs';
 import logger from '../logger.mjs';
 
-
 let service = null;
 
 export function start(config) {
     service = express()
         .get('/', (req, res) => res.send('Hello'))
+        .get('/sessions_update', async (req, res) => {
+            const newSessions = await updateSessions();
```

Comment:
Now I think that we may need some more routes. In this case, we need to implement Controller and UseCase layers. You can add this taking as an example ui service